index.html

```diff
@@ -1,16 +1,102 @@
+            <ul class="navbar">
+                <li><a href=".general">Home</a></li>
+                <li><a href=".cake-pics">Cakes</a></li>               
+                <li><a href="#">Order In</a></li>
+                <li><a href="#">Lessons</a></li>
+                <li><a href=".contact-icon">Contact Us</a></li>
+            </ul>
+            </div>
+        </nav>
+        
+     <div class="name">
+       <h1>Sweet Obsessions</h1>
+       <p class="slogan">The best cakes delivered to your door.</p>
+     </div>
+      
+    </header>
+
+    <article class="general">
+        <summary class="text">
+            <h2>Welcome</h2>
+            <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ratione architecto voluptate earum dolorum, rerum delectus eum omnis accusantium? Sapiente, architecto dignissimos in natus quod aliquam voluptates unde est voluptatum magnam?Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque autem animi eaque? Delectus, voluptatem provident dolor consequatur dignissimos iste? Molestias inventore quam consectetur voluptates expedita quas enim fugit dolorem dolore.</p>
+        </summary>
+        <section class="red-cake"><img class="cake" src="luisana-zerpa-MJPr6nOdppw-unsplash.jpg" alt="red velvet cupcake with cream cheese icing and red velvet sprinkles"></section>
+    </article>
+
+    <article class="cake-pics">
+        <div class="sweet1">
+            <section class="image-sweets" >
+            <img src="daniel-klein-RE2xiNk3KFY-unsplash.jpg" alt="assorted cupcakes in a blue lined box" />
+            </section>
+            <section class="image-sweets"><img src="olivia-oliver-design-8lDEn_W-WAA-unsplash.jpg" alt="mini-cupcakes with pink icing, multicolored sprinkles and a rainbow and cloud topper"></section>
+        </div>
+        <div class="sweet2">
+            <section class="image-sweets"><img src="deva-williamson-m4A2KWBU7xI-unsplash.jpg" alt="multicolored round cake with white chocolate drippings and multicolored pastel icing"></section>
+            <section class="image-sweets">
+            <img src="katie-rosario-QNyRp21hb5I-unsplash.jpg" alt="pink round cake with pink chocolate drippings topped with an upside down ice-cream cone" />
+            </section>
+            <section class="image-sweets">
+            <img src="slashio-photography-GKcoau1ESxg-unsplash.jpg" alt="two cakes, one blue and white and the second is chocolate" />
+            </section>
+        </div>
+    </article>
+
+    <footer class="end">
+        <h2>Contact</h2>
+        <p>Andriana</p>
+        <aside class="contact-icon">
+            <span class="material-symbols-outlined">call</span>
+            <p>075 xxxx xxxx</p>
+        </aside>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+        <aside class="contact-icon">
+            <span class="material-symbols-outlined">mail</span>
+            <p>info@sweetobsessions.com</p>
```

Comment:
This is optional, but you may want to look into wrapping this email in a link so users can click it and automatically start an email. Here is some info on how to do this: https://www.w3docs.com/snippets/html/how-to-create-mailto-links.html